### ISSUE TITLE
Training using long stories instead of question/response

### ISSUE THREAD

**Issue by leszekhanusz** [lxe/simple-llama-finetuner#24]
Is it possible to finetune for stories generation instead of short question/answer like it is possible to do with [NovelAI Custom AI modules](https://blog.novelai.net/custom-ai-modules-dbc527d66081)?

**Comment by lxe** [comment]
I think so... never tried it this way, and will need to play around with the dataset.

**Comment by sanj33vk** [comment]
Great work on the repo @lxe . I wish to finetune for neutralizing the biases in the LLMs. So, instead of training with question/response I want to establish similarity via sentence pairing.

So if i want to establish that two sentence pairs are similar, is it okay to replace the question/response parts of the model to sentence A and sentence B. Is it a good way to proceed?

**Comment by lxe** [comment]
Of course! Use a smaller dataset at first, crank up the learning rate to overfit a bit, then test how it behaves before tuning on larger data
